css/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  background: beige;
+}
+#titlewrap {
+  text-align: center;
+  margin-top: 20px;
+}
+#titlewrap p {
+  padding-top: 3px;
+}
+#survey-form {
+  margin-left: 478px;
+  margin-top: 30px;
+  background: darkgray;
+  width: max-content;
+  font-size: medium;
+  font-style: normal;
+  font-family: initial;
+}
+#submit {
+  height: 30px;
+  width: -webkit-fill-available;
+  background: cornflowerblue;
+}
+.div1 {
+  margin: 0 auto 1.25rem auto;
+  padding: 0.25rem;
+}
+#name,
+#email,
+#number {
+  height: 22px;
+}
+#number {
+  width: 100px;
+}
+#dropdown {
+  height: 22px;
+  width: 100px;
+}
+hr {
+}
```

Comment:
- [ ]  Leaving unused or commented code can make your code unreadable. So. please kindly remove a code that you are not using.